cmd/access/node_builder/access_node_builder.go

```diff
@@ -1839,6 +1848,41 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 			return builder.RequestEng, nil
 		})
 
+	if builder.versionControlEnabled {
+		builder.Component("version control", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+			ver, err := build.Semver()
```

Comment:
best to use explicit variable names, especially when there's limited context on what the short name means
```suggestion
			nodeVersion, err := build.Semver()
```